### PR TITLE
Updated dependencies

### PR DESCRIPTION
**Pull request by alwx**
**Proposed changes**:
It's basically just a result of the recent `dependabot` run combined in 1 PR.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -92,15 +92,15 @@ colorhash = "~1.0.2"
 pika = "~1.1.0"
 jsonschema = "~3.2"
 packaging = ">=19.0,<20.4"
-gevent = ">=1.4,<1.6"
-pytz = "^2019.1"
+gevent = ">=1.4,<20.6"
```
Comment (wochinge) [inline pyproject.toml:95]: haha, the occasional version jump from 1 to 20 🤣

File: pyproject.toml [hunk 2]
```diff
@@ -129,7 +129,7 @@ pytest-cov = "^2.8.1"
 pytest-localserver = "^0.5.0"
 pytest-sanic = "^1.6.1"
 pytest-asyncio = "^0.10.0"
-pytest-xdist = "^1.31.0"
+pytest-xdist = "^1.32.0"
```
Comment (wochinge) [inline pyproject.toml:132]: This means `1.31.0` is now incompatible?

File: pyproject.toml (hunk 1 above)
Comment (alwx) [reply pyproject.toml:95]: Yes, they simply switched from the normal naming to something like 20.4.0 where 20 is a year and 4 is a month.
https://pypi.org/project/gevent/#history

File: pyproject.toml (hunk 2 above)
Comment (alwx) [reply pyproject.toml:132]: That's a dev dependency, so it doesn't matter — we simply update dev- dependencies to the latest versions if possible.

**Pull request merged**